src/xrGame/ui/UIMapWnd.h

```diff
@@ -156,10 +156,10 @@ class CUIMapWnd : public CUIWindow, public CUIWndCallback
     };
     void AddMapToRender(CUICustomMap*);
     void RemoveMapToRender(CUICustomMap*);
-    CUIGlobalMap* GlobalMap() { return m_GlobalMap; };
-    const GameMaps& GameMaps() { return m_GameMaps; };
+    CUIGlobalMap* GlobalMap() { return m_GlobalMap; }
+//    const GameMaps& GetGameMaps() { return m_GameMaps; }
```

Comment:
Why commented? Unused?
Then, it's better to comment it like this:
```suggestion
    //const GameMaps& GetGameMaps() { return m_GameMaps; }
```

Reply:
It is better to remove the comment, I think